project_timesheet_time_control/models/account_analytic_line.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2016 Tecnativa - Sergio Teruel
+# Copyright 2016-2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+    _order = "date_time desc"
```

Comment:
I'm always reluctant to change standard `_order` as it can cause problems in side modules.

Reply:
@jdebetaz Agree with that, maybe you could find in the code where date_time desc is needed and use a sorted instead?
do it in another commit in case backport is interesting ?

Reply:
This is needed because we are introducing this new field with datetime precision, so if not done, timesheets from the same day but different time will be unsorted. It should be declared at model level. It shouldn't be changed.